prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,5930 @@
+                            }
+                          ],
+                          "code": "EN-US"
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "url": "MSH.20",
+                    "valueString": "UTF-16"
+                  },
+                  {
+                    "url": "MSH.21",
+                    "valueIdentifier": {
+                      "extension": [
+                        {
+                          "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
+                          "extension": [
+                            {
+                              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                              "valueString": "2.16.840.1.113883.9.82"
+                            },
+                            {
+                              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                              "valueCode": "ISO"
+                            }
+                          ]
+                        }
+                      ],
+                      "value": "LAB_PRU_COMPONENT"
+                    }
+                  },
+                  {
+                    "url": "MSH.24",
+                    "extension": [
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
+                        "valueString": "Alt Sending App"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                        "valueString": "Alt Sending App Universal"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                        "valueString": "UUID"
+                      }
+                    ]
+                  },
+                  {
+                    "url": "MSH.25",
+                    "extension": [
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
+                        "valueString": "Sending App"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                        "valueString": "Sending App Universal"
+                      },
+                      {
+                        "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                        "valueString": "ISO"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      Extension:
+        - id: sender-id
```

Comment:
I think this is missing in the example above?  But also thinking that maybe we cannot include it, I think we only use this internally

Reply:
good call. Removed